pkgs/by-name/au/authenticator-rs/fix-install.patch

```diff
@@ -0,0 +1,133 @@
+diff --git a/Makefile b/Makefile
+index 978f151..a177152 100644
+--- a/Makefile
++++ b/Makefile
+@@ -41,15 +41,13 @@ release-version:
+ 	sed -i 's/#VERSION_NUMBER#/$(RELEASE_VERSION)/' ./data/resources/gtk/ui/main.ui
+ 
+ install-po: # dev only - run with sudo
+-	msgfmt po/fr.po -o /usr/share/locale/fr/LC_MESSAGES/authenticator-rs.mo
+-	msgfmt po/en_GB.po -o /usr/share/locale/en_GB/LC_MESSAGES/authenticator-rs.mo
++	msgfmt po/fr.po -o $(sharedir)/locale/fr/LC_MESSAGES/authenticator-rs.mo
++	msgfmt po/en_GB.po -o $(sharedir)/locale/en_GB/LC_MESSAGES/authenticator-rs.mo
+ 
+ # Install onto the system
+-install : release gresource
++install : gresource
+ 	# Create the bindir, if need be
+ 	mkdir -p $(bindir)
+-	# Install binary
+-	$(INSTALL_PROGRAM) target/release/authenticator-rs $(bindir)/authenticator-rs
+ 
+ 	# Create the sharedir and subfolders, if need be
+ 	mkdir -p $(sharedir)/icons/hicolor/scalable/apps/
+@@ -62,7 +60,7 @@ install : release gresource
+ 
+ 	# Install gResource
+ 	$(INSTALL_DATA) data/uk.co.grumlimited.authenticator-rs.gresource $(sharedir)/uk.co.grumlimited.authenticator-rs/uk.co.grumlimited.authenticator-rs.gresource
+-	
++
+ 	# Install icons
+ 	$(INSTALL_DATA) data/icons/hicolor/scalable/apps/uk.co.grumlimited.authenticator-rs.svg $(sharedir)/icons/hicolor/scalable/apps/uk.co.grumlimited.authenticator-rs.svg
+ 	$(INSTALL_DATA) data/icons/hicolor/64x64/apps/uk.co.grumlimited.authenticator-rs.64.png $(sharedir)/icons/hicolor/64x64/apps/uk.co.grumlimited.authenticator-rs.png
+@@ -71,7 +69,7 @@ install : release gresource
+ 	# Force icon cache refresh
+ 	touch $(sharedir)/icons/hicolor
+ 
+-	# Install application meta-data
++	# Install application metadata
+ 	$(INSTALL_DATA) data/uk.co.grumlimited.authenticator-rs.appdata.xml $(sharedir)/metainfo/uk.co.grumlimited.authenticator-rs.appdata.xml
+ 
+ 	# Install desktop file
+@@ -79,12 +77,11 @@ install : release gresource
+ 
+ 	# Install gschema file
+ 	$(INSTALL_DATA) data/uk.co.grumlimited.authenticator-rs.gschema.xml $(sharedir)/glib-2.0/schemas/
+-	
++
+ 	# Install LOCALE files
+ 	rm -fr builddir/
+-	meson builddir --prefix=/usr
++	meson setup --prefix=$(PREFIX) builddir
+ 	DESTDIR=../build-aux/i18n meson install -C builddir
+-	cp -fr build-aux/i18n/usr $(DESTDIR)
```

Comment:
Why was this removed?

Reply:
This is removed in https://github.com/grumlimited/authenticator-rs/pull/138 in order to have the `meson install` line do the copy itself. We need to do that since the current working directory doesn't have `build-aux` when invoked by Nix.